forward_checking.py

```diff
@@ -0,0 +1,63 @@
+from AI_Sudoku.sudoku import Sudoku
+from AI_Sudoku.field import Field
```

Comment:
```suggestion
from field import Field
```

Reply:
I don't think it's neccesary to add AI_Sudoku at imports. It works perfectly fine without them.